examples/vision/semisupervised_simclr.py

```diff
@@ -199,24 +200,25 @@ def get_config(self):
 
     def call(self, images, training=True):
         if training:
-            batch_size = tf.shape(images)[0]
+            batch_size = ops.shape(images)[0]
 
             # Same for all colors
-            brightness_scales = 1 + tf.random.uniform(
+            brightness_scales = 1 + keras.random.uniform(
                 (batch_size, 1, 1, 1),
                 minval=-self.brightness,
                 maxval=self.brightness,
```

Comment:
Please add a `seed=self.seed_generator` arg and create `self.seed_generator` in `__init__()`.